# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpe-autostarter/linux-header.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gpe-autostarter-0.12/net.c.vanilla	2007-08-14 17:36:08.000000000 +0200
+++ gpe-autostarter-0.12/net.c	2007-08-14 17:41:19.000000000 +0200
@@ -19,6 +19,8 @@
 #include <signal.h>
 
 #include <sys/socket.h>
+#include <linux/types.h>
+#include <linux/if.h>
 #include <linux/wireless.h>
 #include <sys/ioctl.h> 
 
